wordplay.py

```diff
@@ -0,0 +1,244 @@
+for word in words: 
+    vowel_in_word = True
+    for char in vowels:
+        if char not in word:
+            vowel_in_word = False
+            break
+    if vowel_in_word:
+        words_with_all_vowels.append(word)
+# To Display Answer
+# print(words_with_all_vowels)
+
+#Question: 
+# What are all of the words that have all 5 vowels, in alphabetical order? 
+words_with_alphabetical_vowels = []
+vowels = "AEIOU"
+
+for word in words:
+    word_index = 0
+    vowel_index = 0
+    while word_index < len(word) and vowel_index < len(vowels):
+        if word[word_index] == vowels[vowel_index]:
+            vowel_index += 1
+        word_index += 1
+    if vowel_index == len(vowels):
+            words_with_alphabetical_vowels.append(word)
+#to display answer:
+#print(words_with_alphabetical_vowels)
+
+
+#Question: 
+#How many words contain the substring "TYPE”?
+type_count = 0
+for word in words:
+    if "TYPE" in word:
+        type_count += 1
+# To display an answer
+# print(type_count)
+
+#Question: 
+#Create and print an array containing all of the words that end in "GHTLY"
+end_in_GHTYL = []
+for word in words:
+    if word[-5:] ==  "GHTLY":
+         end_in_GHTYL.append(word)
+#To display all the words
+#print(end_in_GHTYL)
+
+#Question:
+#What is the shortest word that contains all 5 vowels?
+
+shortest_word_all_vowels = words_with_all_vowels[0]
+for word in words_with_all_vowels:
+    if len(word) < len(shortest_word_all_vowels):
+        shortest_word_all_vowels = word
+# To display answer:
+#print(shortest_word)
+
+
+
+#What is the longest word that contains no vowels? Assumes Y is allowed. 
+
+vowels = "AEIOU"
+words_no_vowels = []
+for word in words:
+    vowel_in_word = False
+    for char in vowels:
+        if char in word:
+            vowel_in_word = True
+            break 
+    if not vowel_in_word:
+        words_no_vowels.append(word)
+
+longest_word_no_vowels = words_no_vowels[0]
+for word in words_no_vowels:
+    if len(word) > len(longest_word_no_vowels):
+        longest_word_no_vowels = word
+
+#To display Answer
+#print(longest_word)
+    
+
+#Question
+#Which of the letters Q, X, and Z is the least common?
+letter_Q_count = 0
+letter_X_count = 0
+letter_Z_count = 0
+
+for word in words:
+    for char in word:
+        if "Q" == char:
+            letter_Q_count += 1
+        if "X" == char:
+            letter_X_count += 1
+        if "Z" == char:
+            letter_Z_count += 1
+#To Display Answer: 
+# if letter_Q_count and letter_X_count > letter_Z_count:
+#     print('letter Z count:', letter_Z_count)
+# elif letter_Z_count and letter_X_count > letter_Q_count:
+#     print('letter Q count:', letter_Q_count)
+# else:
+#     print('letter X count:', letter_X_count)
+
+#  Question: answered same question as above but used dictionary
+dic = {"X":0, "Q":0, "Z":0}
+for word in words:
+    for char in word:
+        if char in dic:
+            dic[char] += 1
+min_letter = ""
+min_count = float('inf')  
```

Comment:
Nice!

I think using `float('inf')` as your default comparator here is a great move - if you just pick a large number you run the risk of making a choice that leads to bugs while `float('inf') > x` will always evaluate to `True`